docs/designers-developers/developers/filters/block-filters.md

```diff
@@ -237,8 +237,9 @@ Adding blocks is easy enough, removing them is as easy. Plugin or theme authors
 
 ```js
 // my-plugin.js
-
-wp.blocks.unregisterBlockType( 'core/verse' );
+_wpLoadBlockEditor.then( function() {
+	wp.blocks.unregisterBlockType( 'core/verse' );
+});
```

Comment:
CS: missing space.

```suggestion
} );
```

Reply:
Does `function()` also need to be `function ()` or am I confusing my coding styles? 😅

Reply:
`function()` is fine. Although the JS coding standards don't contain [an example](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#arrays-and-function-calls), the actual lint rule [applied](https://github.com/WordPress/gutenberg/blob/6c688159babfbea1adfa2eda536066cd19762813/packages/eslint-config/index.js#L116) ([space-before-function-paren](https://eslint.org/docs/rules/space-before-function-paren)) is configured to only require space in async arrow functions.